Formula/curl.rb

```diff
@@ -1,8 +1,9 @@
 class Curl < Formula
   desc "Get a file from an HTTP, HTTPS or FTP server"
   homepage "https://curl.se"
+  # Don't forget to update both instances of the version in the GitHub mirror URL.
   url "https://curl.se/download/curl-8.1.2.tar.bz2"
-  mirror "https://github.com/curl/curl/releases/download/curl-8_1_1/curl-8.1.2.tar.bz2"
+  mirror "https://github.com/curl/curl/releases/download/curl-8_1_2/curl-8.1.2.tar.bz2"
```

Comment:
We should ideally add a test for this, I think.

Reply:
Added a test to match the tag name.